sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_region.c

```diff
@@ -21,6 +21,7 @@
 #include "debug.h"
 
 #define GNRC_LORAWAN_DATARATES_NUMOF (6U)
+#define GNRC_LORAWAN_TX_POWER_NUMOF  (8U)
```

Comment:
I think it would be more appropiate to reflect "MAX POWER" rather than "NUMOF"

Reply:
ping?